From 781ec2cfecacb38b354dc0757acda235ff79928e Mon Sep 17 00:00:00 2001
From: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Date: Sun, 18 Jul 2021 15:07:38 +0300
Subject: [BUGFIX][NOUPSTREAM] iwlwifi: iwlmei can't work on 5.14

Update the Kconfig accordingly.

type=bugfix
ticket=none
fixes=Ibf66e3e248d549166d1eb340c60ee5b2a1dada78

Change-Id: I04f5878f8f8d85f0f3c56cb0d7251e64df07a6db
Signed-off-by: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Reviewed-on: https://git-amr-3.devtools.intel.com/gerrit/323487
Reviewed-by: Johannes Berg <johannes.berg@intel.com>
automatic-review: ec ger unix iil jenkins <EC.GER.UNIX.IIL.JENKINS@INTEL.COM>
Tested-by: ec ger unix iil jenkins <EC.GER.UNIX.IIL.JENKINS@INTEL.COM>
x-iwlwifi-stack-dev: 4ea5eedf88c6ed5cbdbe04b871a9787f4617e9a7
(backported from commit 8e2e67754e3e5d2819c4071b9b3d5a2ba890e9b4)
Signed-off-by: You-Sheng Yang <vicamo.yang@canonical.com>
---
 drivers/net/wireless/intel/iwlwifi/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/intel/iwlwifi/Kconfig b/drivers/net/wireless/intel/iwlwifi/Kconfig
index 350ea2986..da9e5935b 100644
--- a/drivers/net/wireless/intel/iwlwifi/Kconfig
+++ b/drivers/net/wireless/intel/iwlwifi/Kconfig
@@ -93,7 +93,7 @@ config IWLMEI
 	depends on INTEL_MEI
 	depends on IWLMVM
 	depends on IWLMVM_VENDOR_CMDS
-	depends on WLAN_VENDOR_INTEL_MEI || !KERNEL_5_14
+	depends on WLAN_VENDOR_INTEL_MEI || !KERNEL_5_15
 	help
 	  Enable V-PRO. This allows to communicate with the CSME firmware.
 	  This is supported starting from Tiger Lake platforms and has been
-- 
2.34.1

